packages/core/src/classes.ts

```diff
@@ -59,35 +59,56 @@ export type AllAttributes<T extends E, E extends Obj> = Required<Attributes<T, E
 
 export enum ClassifierKind {
   CLASS,
-  MIXIN
+  MIXIN,
+  ENUM
 }
 
-export interface EClassifier<T extends E, E extends Obj> extends Doc {
+export interface Classifier extends Doc {
   _kind: ClassifierKind
-  _attributes: AllAttributes<T, E>
-  _extends?: Ref<Classifier<E>>
 }
 
-export type Classifier<T extends Obj> = EClassifier<T, Obj>
-
 export interface EMixin<T extends E, E extends Obj> extends EClass<T, E> {
 }
 
 export type Mixin<T extends Obj> = EMixin<T, Obj>
 
-export interface EClass<T extends E, E extends Obj> extends EClassifier<T, E> {
-  _native?: StringProperty
+export interface EDomainClassifier {
   _domain?: StringProperty
 }
```

Comment:
I guess `domain` is only Class property, since it's related to storage options. Do we really need `domain` for enums?

Reply:
Good catch, we are not need it, thanks.